packages/commercetools/theme/components/Checkout/VsfShippingProvider.vue

```diff
@@ -6,15 +6,17 @@
       class="sf-heading--left sf-heading--no-underline title"
     />
     <div class="form">
-      <div v-if="error.loadMethods">
-        {{ $t('There was some error while trying to fetch shipping methods. We are sorry, please try with different shipping details or later.') }}
-      </div>
-      <div v-else-if="errorShippingProvider.save">
-        {{ $t('There was some error while trying to select this shipping method. We are sorry, please try with different shipping method or later.') }}
-      </div>
-      <div v-else-if="!shippingMethods.length">
-        {{ $t('There are no shipping methods available for this country. We are sorry, please try with different country or later.') }}
-      </div>
+      <SfLoader :loading="loading">
+        <div v-if="error.loadMethods">
+          {{ $t('There was some error while trying to fetch shipping methods. We are sorry, please try with different shipping details or later.') }}
+        </div>
+        <div v-else-if="errorShippingProvider.save">
+          {{ $t('There was some error while trying to select this shipping method. We are sorry, please try with different shipping method or later.') }}
+        </div>
+        <div v-else-if="!shippingMethods.length">
+          {{ $t('There are no shipping methods available for this country. We are sorry, please try with different country or later.') }}
+        </div>
+      </SfLoader>
       <div class="form__radio-group">
```

Comment:
shouldn't this also be conditionally visible?

Reply:
It wasn't conditionally visible before but I can make it, i.e. create `hasErrors` computed that will return true if there are any errors and add `v-if="hasErrors"` to `SfLoader` and `v-else` to this `div`. What do you think?